getZoraProfile.ts

```
@@ -0,0 +1,6 @@
+async function getZoraProfile(address: string) {
```

ESLint

[error] 'getZoraProfile' is defined but never used.

(@typescript-eslint/no-unused-vars)